### ISSUE TITLE
Doesn't build on i386

### ISSUE THREAD

**Issue by trebmuh** [brummer10/Xmonk.lv2#5]
Hi @brummer10 , trying to build on a Debian Buster on a i386 chroot, it gives me the following:

```
g++ -g -O2 -fdebug-prefix-map=/build/xmonk.lv2-0.4=. -fstack-protector-strong -Wformat -Werror=format-security -D_FORTIFY_SOURCE=2 -I. -I./dsp -I./plugin -fPIC -DPIC -O2 -Wall -funroll-loops -fstack-protector -ffast-math -fomit-frame-pointer -fstrength-reduce -fdata-sections -Wl,--gc-sections -Wl,-z,relro,-z,now -Wl,--exclude-libs,ALL -msse3 -mfpmath=sse plugin/Xmonk.cpp  -Wl,-z,relro -I. -shared -lm -Wl,-z,noexecstack -fvisibility=hidden -o Xmonk.so
plugin/Xmonk.cpp: In constructor 'xmonk::DenormalProtection::DenormalProtection()':
plugin/Xmonk.cpp:121:5: error: '__builtin_ia32_fxsave' was not declared in this scope
     __builtin_ia32_fxsave(&fxsave);
     ^~~~~~~~~~~~~~~~~~~~~
plugin/Xmonk.cpp:121:5: note: suggested alternative: '__builtin_ia32_xsave'
     __builtin_ia32_fxsave(&fxsave);
     ^~~~~~~~~~~~~~~~~~~~~
     __builtin_ia32_xsave
make[2]: *** [Makefile:99: Xmonk] Error 1
make[2]: Leaving directory '/build/xmonk.lv2-0.4/Xmonk'
make[1]: *** [Makefile:18: Xmonk] Error 2
make[1]: Leaving directory '/build/xmonk.lv2-0.4'

```

Any idea maybe?

**Comment by brummer10** [comment]
Could you try please with` -mfxsr` added to the CXXFLAGS in the makefile.
I don't have a i386 box,  so I only guessing.

**Comment by trebmuh** [comment]
That did the trick.
I mean, with adding `-mfxsr` to the `CXXFLAGS` in `Xmonk/Makefile`, it built on a Debian Buster i386 chroot here, note that I didn't test the resulting binary on a i386 system, just it built (tm).
Thank you very much @brummer10 , closing here now.

**Comment by brummer10** [comment]
Thank you for trying. So I could add a fix for this in general to all affected projects. 
btw. you are aware of this one:  https://github.com/brummer10/GxMatchEQ.lv2
as it use the same denormalize technic then this one here.

**Comment by trebmuh** [comment]
I'm not! Usually, I'm only using/packaging Guitarix and the GXplugins suite (plus screcord, gxtuner, and jalv.select). Is this GXmatchEQ supposed to be incorporated into the GXplugins at a later point? The reason I'm asking is because, I would probably avoid to package it for LZK if it is expected to join the GXplugins family (it's a bit of a pain to manage this king of transition).
BTW: do you have other plugins/project LA-related maybe?

**Comment by brummer10** [comment]
No, it will not join the GxPlugins suite. all GxPlugins been suited for the MOD, that is not the case for GxMatchEQ. 
I've some other projects LA related, but nothing worse to package. So when you take that one, you'll have all worse to keep a eye on it. Note, for now it will have the same issue as this one here, I need to push a fix latter on.

**Comment by trebmuh** [comment]
OK, thanks for the time you took for that. I'll look at packaging this GxMatchEQ for LZK soon-ish.

**Comment by trebmuh** [comment]
Hi @brummer10 , just came back reading at this one.
It does seem that the "mfxsr" thingy has not made it into the master code here. Is that on purpose?

**Comment by brummer10** [comment]
Yea, I've forgotten about it. Should be fixed now with the latest commit.
